apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+/* eslint complexity: 0 max-statements: 0 */
+
+module.exports = data => {
+  const response = {};
+
+  response.Type = '560734';
+  response['Customer.FirstName'] = data['pv-name-first-name'];
+  response['Customer.Surname'] = data['pv-name-last-name'];
+  response['Customer.Address'] = data['pv-contact-details-street'];
+  response['Customer.Town'] = data['pv-contact-details-town'];
+  response['Customer.County'] = data['pv-contact-details-county'];
+  response['Customer.Postcode'] = data['pv-contact-details-postcode'];
+  response['Customer.ContactMethod'] = data['pv-contact-details'];
+  response['Customer.Alias'] = data['pv-name-nickname'];
+  if (data['pv-dob']) {
+    response['Customer.DOBKnown'] = 'Yes';
+    response['Customer.DOB'] = data['pv-dob'];
+  }
+  // response['Customer.AgeGroup'];
+  const genderMap = {
+    'female': 'Female',
+    'male': 'Male',
+    'unknown': 'Other'
+  };
+  response['Customer.iGender'] = genderMap[data['pv-gender']];
+  response['Customer.Email'] = data['pv-contact-details-email-input'];
+  response['Customer.Mobile'] = data['pv-phone-number-yes'];
+  response['Customer.Mobile'] = data['pv-phone-number-yes'];
+  // response['Customer.ContactTime'];
+  response['Customer.HaveChildren'] = data['does-pv-have-children'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.NumberOfChildren'] = data['does-pv-have-children-yes-amount'];
+  response['Customer.Nationality'] = data['pv-nationality'];
+  response['Customer.DualNationality'] = data['pv-nationality-second'];
+  // response['Customer.Country'];
+  // response['Customer.ReferenceNumbers'] = data['pv-ho-reference-type'];
+  response['Customer.InterpreterNeeded'] = data['pv-interpreter-requirements'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.InterpreterLanguage'] = data['pv-interpreter-requirements-language'];
+  response['Customer.HelpWithCommunication'] = data['pv-other-help-with-communication'] === 'yes' ? 'Yes' : 'No';
+  response['Customer.CommunicationAid'] = data['pv-other-help-with-communication-aid'];
+  // response['Customer.PassportNumber'];
```

Comment:
comment?

Reply:
yes because it's in the API but we don't know what to do with it yet